src/main/java/helper/SuccessResponseGenerator.java

```diff
@@ -182,4 +182,18 @@
         
         return getResponseBodyForListMessages(listOfMessagesImmutable);
     }
+
+    /**
+     * Renders the given parameters in a Map to return a successful HTTP response for all client requests to the GetAttachment API.
+     */
+    public static Map<String, Object> getSuccessResponseForGetAttachment(Attachment attachment, byte[] blob) {
```

Comment:
The return type here should be `ImmutableMap`.

Reply:
Yes, done.